tests/unit/nupic/frameworks/opf/opf_metrics_test.py

```diff
@@ -47,6 +47,19 @@ def testRMSE(self):
 < OPFMetricsTest.DELTA)
 
 
+  def testNRMSE(self):
+    nrmse = getModule(MetricSpec("nrmse", None, None,
+{"verbosity" : OPFMetricsTest.VERBOSITY}))
+    gt = [9, 4, 5, 6]
+    p = [0, 13, 8, 3]
+    for i in xrange(len(gt)):
+      nrmse.addInstance(gt[i], p[i])
+    target = 1.342
+
+    self.assertTrue(abs(nrmse.getMetric()["value"]-target)\
```

Comment:
Why did you break the line? It looks like it is under 80 characters without the break and it is inside parens so no need for backslash anyway